examples/src/main/scala/example/akkahttp/ExampleApp.scala

```diff
@@ -18,9 +18,9 @@ import zio.Runtime
 
 object ExampleApp extends App {
 
-  implicit val system: ActorSystem                                      = ActorSystem()
-  implicit val executionContext: ExecutionContextExecutor               = system.dispatcher
-  implicit val runtime: Runtime[ExampleService with Console with Clock] =
+  implicit val system: ActorSystem                                              = ActorSystem()
+  implicit val executionContext: ExecutionContextExecutor                       = system.dispatcher
+  implicit val runtime: Runtime.Managed[ExampleService with Console with Clock] =
```

Comment:
Any reason to change that?

Reply:
Yeah, the `shutdown` method is only available on the `Runtime.Managed`, and I am terminating it now when the server shuts down